backend/Tasque.BLL/Services/EntityCrudService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Tasque.Core.Common.Entities.Abstract;
+using Tasque.Core.DAL;
+
+namespace Tasque.Core.BLL.Services
+{
+    public abstract class EntityCrudService<TEntity> where TEntity : BaseEntity
+    {
+        private readonly DataContext _db;
+        public EntityCrudService(DataContext db)
+        {
+            _db = db;
+        }
+
+        public TEntity Create(TEntity entity)
+        {
+            _db.Set<TEntity>().Add(entity);
+            _db.SaveChanges();
+            return entity;
+        }
+
+        public TEntity? ReadOne(int id)
```

Comment:
I know that R in CRUD stands for Read but conventionally we use Get for these kinds of methods
GetById, GetAll, etc

Reply:
Renamed in both service and controller